shm/test.cpp

```diff
@@ -0,0 +1,45 @@
+#include "cargo.hpp"
+#include "coordinates.hpp"
+#include "island.hpp"
+#include "map.hpp"
+#include "ship.hpp"
+#include "player.hpp"
+
+#include <vector>
+#include "gtest/gtest.h"
+
+
+class SHM_TestFixture: public ::testing::Test {
+public:
+    SHM_TestFixture( ) {
+        // initialization code here
+    }
```

Comment:
This is ctor for test, where you can initialize important values, which should be implemented always in all tests using this fixture.